app/src/main/java/com/peacecorps/pcsa/LoginActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.peacecorps.pcsa;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.Toast;
+
+public class LoginActivity extends AppCompatActivity  implements AdapterView.OnItemSelectedListener {
+
+    private Toolbar toolbar;
+    private Button loginButton;
+    private EditText name;
+    private Spinner country;
+    private String selected_country;
+    SharedPreferences.Editor editor;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+
+        editor = PreferenceManager.getDefaultSharedPreferences(LoginActivity.this).edit();
```

Comment:
No, this is a different variable.
Also, I think we cannot use the same variable used in Circle of Trust because that variable checks for first time run of Circle of Trust and not the app itself.

However, like you mentioned there is absolutely no point in having this variable when I can detect it using the variables created for country and username